gui-apps/xwaylandvideobridge/xwaylandvideobridge-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.83.0
+QTMIN=5.15.0
+KDE_ORG_CATEGORY="system"
+
+inherit ecm kde.org xdg
+
+DESCRIPTION="Screenshare Wayland windows to XWayland apps"
+
+if [[ ${KDE_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+
+DEPEND="
+	>=dev-qt/qtdbus-${QTMIN}
+	>=dev-qt/qtquickcontrols-${QTMIN}
+	>=dev-qt/qtx11extras-${QTMIN}
+	>=kde-frameworks/kcoreaddons-${KFMIN}
+	>=kde-frameworks/ki18n-${KFMIN}
+	>=kde-frameworks/kwindowsystem-${KFMIN}
+	>=kde-frameworks/knotifications-${KFMIN}
+	>=kde-frameworks/kwidgetsaddons-${KFMIN}
+	>=kde-plasma/kpipewire-5.27.4
+	media-libs/freetype
+	x11-libs/libxcb
```

Comment:
should it have a slot op?

Reply:
Yes indeed it should, completely forgot to do the slot operators here.